src/commands/prs.js

```diff
@@ -45,47 +45,46 @@ exports.handler = async ({ templates: { title, body } }, state) => {
 
 	const octokit = getOctokit(githubAccessToken)
 
-	await promiseAllErrors(
-		state.repos.map(async repo => {
-			if (repo.remoteBranch) {
-				const [existingPr] = await octokit.paginate(
-					octokit.pulls.list.endpoint.merge({
-						owner: repo.owner,
-						repo: repo.name,
-						head: `${repo.owner}:${repo.remoteBranch}`,
-					}),
-				)
+	for (const repo of state.repos) {
+		if (repo.remoteBranch) {
+			const [existingPr] = await octokit.paginate(
+				octokit.pulls.list.endpoint.merge({
+					owner: repo.owner,
+					repo: repo.name,
+					head: `${repo.owner}:${repo.remoteBranch}`,
+				}),
+			)
 
-				if (existingPr) {
-					logger.log(`${repo.owner}/${repo.name} PR`, {
-						status: 'info',
-						message: `using existing PR ${styles.url(
-							existingPr.html_url,
-						)} for ${styles.branch(repo.remoteBranch)} on ${styles.repo(
-							`${repo.owner}/${repo.name}`,
-						)}`,
-					})
-				}
-				repo.pr =
-					existingPr ||
-					(await logger
-						.logPromise(
-							octokit.pulls.create({
-								owner: repo.owner,
-								repo: repo.name,
-								head: repo.remoteBranch,
-								base: repo.centralBranch,
-								title: titleTemplate(repo),
-								body: bodyTemplate(repo),
-							}),
-							`creating PR for ${styles.branch(
-								repo.remoteBranch,
-							)} on ${styles.repo(`${repo.owner}/${repo.name}`)}`,
-						)
-						.then(response => response.data))
+			if (existingPr) {
+				logger.log(`${repo.owner}/${repo.name} PR`, {
+					status: 'info',
+					message: `using existing PR ${styles.url(
+						existingPr.html_url,
+					)} for ${styles.branch(repo.remoteBranch)} on ${styles.repo(
+						`${repo.owner}/${repo.name}`,
+					)}`,
+				})
 			}
-		}),
-	)
+			repo.pr =
+				existingPr ||
+				(await logger
+					.logPromise(
+						octokit.pulls.create({
+							owner: repo.owner,
+							repo: repo.name,
+							head: repo.remoteBranch,
+							base: repo.centralBranch,
+							title: titleTemplate(repo),
+							body: bodyTemplate(repo),
+						}),
+						`creating PR for ${styles.branch(
+							repo.remoteBranch,
+						)} on ${styles.repo(`${repo.owner}/${repo.name}`)}`,
+					)
+					.then(response => response.data))
+			await new Promise(r => setTimeout(r, 2000))
```

Comment:
**nitpick:** maybe we should move this up to be a top level constant that we can tweak rather than a magic number (I assume 2 seconds is an arbitrary wait time?)

Reply:
yeah, 2 is a magic number 😅